### PR TITLE
HCIDOCS-413: IPI install no longer requires disk reformat

### PR DESCRIPTION
**Pull request by johnwilkins**
Removed redundant module, included it in the installation assembly, moved Troubleshooting right after installation, and incorporated Vale linter suggestions.

Fixes: [HCIDOCS-413](https://issues.redhat.com//browse/HCIDOCS-413)

See https://issues.redhat.com/browse/HCIDOCS-413 for additional details.

Preview URL: https://81522--ocpdocs-pr.netlify.app/
https://81522--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installing-a-cluster.html
https://81522--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-troubleshooting.html

For release(s): 4.14-4.17
QE Review: 

- [x] QE has approved this change. 

Signed-off-by:  <jowilkin@redhat.com>

### CODE REVIEW

File: installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc [hunk 1]
```diff
@@ -9,19 +9,19 @@ toc::[]
 
 == Troubleshooting the installation program workflow
 
-Prior to troubleshooting the installation environment, it is critical to understand the overall flow of the installer-provisioned installation on bare metal. The diagrams below provide a troubleshooting flow with a step-by-step breakdown for the environment.
+Before troubleshooting the installation environment, it is critical to understand the overall flow of the installer-provisioned installation on bare metal. The following diagrams illustrate a troubleshooting flow with a step-by-step breakdown for the environment.
```
[the remark this answers is not in the document]
Comment (johnwilkins) [reply installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc:12]: Not an error. It is looking for the string "the installer", but ignored the hyphen.

File: installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc [hunk 2]
```diff
@@ -9,19 +9,19 @@ toc::[]
...
 
 image:flow3.png[Flow-Diagram-3]
 
-_Workflow 3 of 4_ illustrates a troubleshooting workflow for xref:ipi-install-troubleshooting-cluster-nodes-will-not-pxe_ipi-install-troubleshooting[ cluster nodes that will not PXE boot]. If installing using RedFish Virtual Media, each node must meet minimum firmware requirements for the installation program to deploy the node. See *Firmware requirements for installing with virtual media* in the *Prerequisites* section for additional details.
+_Workflow 3 of 4_ illustrates a troubleshooting workflow for xref:ipi-install-troubleshooting-cluster-nodes-will-not-pxe_ipi-install-troubleshooting[ cluster nodes that will not PXE boot]. If installing using Redfish virtual media, each node must meet minimum firmware requirements for the installation program to deploy the node. See *Firmware requirements for installing with virtual media* in the *Prerequisites* section for additional details.
```
Comment (mburke5678) [inline installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc:24]: Probably doesn't matter, but still....
```suggestion
_Workflow 3 of 4_ illustrates a troubleshooting workflow for xref:ipi-install-troubleshooting-cluster-nodes-will-not-pxe_ipi-install-troubleshooting[cluster nodes that will not PXE boot]. If installing using Redfish virtual media, each node must meet minimum firmware requirements for the installation program to deploy the node. See *Firmware requirements for installing with virtual media* in the *Prerequisites* section for additional details.
```

**Comment by mburke5678** [comment]
@johnwilkins A couple of comments. Otherwise LGTM.

File: modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc [hunk 3]
```diff
@@ -6,20 +6,20 @@
 [id="ipi-install-troubleshooting-cleaning-up-previous-installations_{context}"]
 = Cleaning up previous installations
 
-In the event of a previous failed deployment, remove the artifacts from the failed attempt before attempting to deploy {product-title} again.
+In case of an earlier failed deployment, remove the artifacts from the failed attempt before trying to deploy {product-title} again.
 
 .Procedure
 
-. Power off all bare metal nodes prior to installing the {product-title} cluster:
+. Power off all bare-metal nodes before installing the {product-title} cluster:
```
[the remark this answers is not in the document]
Comment (johnwilkins) [reply modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc]: Fixed.

**Pull request merged**